cloud-infrastructure/storage/block-storage/README.md

```diff
@@ -0,0 +1,84 @@
+  - Short Videos to introduce Oracle Cloud Infrastructure Block Volume
+- [OCI Block Storage on oracle.com](https://www.oracle.com/cloud/storage/block-volumes/)
+  - Oracle Cloud Infrastructure (OCI) Block Volumes provide reliable, high-performance, low-cost block storage. OCI Block Volumes persist beyond the lifespan of a virtual machine, have built-in redundancy, and can scale to 1 PB per compute instance.
+- [Blogs on Block Storage](https://blogs.oracle.com/authors/max-verun)
+  - See all Block Storage Blogs from Oracle's Product Management
+- [Block Storage Release Notes](https://docs.oracle.com/en-us/iaas/releasenotes/services/blockvolume/)
+- [Block Volumes FAQ](https://www.oracle.com/cloud/storage/block-volumes/faq)
+- [Oracle Cloud Infrastructure Service Level Agreement (SLA)](https://www.oracle.com/cloud/sla/)
+  - Enterprises demand more than just availability from their cloud infrastructure. Mission-critical workloads also require consistent performance, and the ability to manage, monitor, and modify resources running in the cloud at any time. Only Oracle offers end-to-end SLAs covering performance, availability, and manageability of services.
+- [Oracle PaaS and IaaS Public Cloud Services Pillar Document Block Storage](https://www.oracle.com/assets/paas-iaas-pub-cld-srvs-pillar-4021422.pdf#page=28)
+  - Oracle offers end-to-end SLAs covering performance, availability, and manageability. Here you find details regarding Block Storage.
+- [Oracle PaaS and IaaS Universal Credits Service Descriptions - Storage section](https://www.oracle.com/us/corporate/contracts/paas-iaas-universal-credits-3940775.pdf#page=178)
+  - OCI Block Storage Service Description
+- [Resilience and Continuous Availability of Oracle Cloud Infrastructure Services and Platform FAQ](https://www.oracle.com/cloud/iaas/faq.html)
+  - Resilience and Continuous Availability FAQs
+- [Oracle Cloud Infrastructure Vault: Block Volume Encryption](https://www.youtube.com/watch?v=3GBPIx4hlRU)
+ - This video helps you to encrypt a block volume with the KMS Vault customer-managed encryption key.
+
+## Reusable Assets Overview
+
+- [Cloud Resilience](https://gitlab.com/hmielimo/cloud-resilience/-/blob/main/doc/cloud.resilience/README.md)
+  - All details regarding storage based resiliency incl. example backup & recover automation script
+- [Cloud Resilience by default](https://gitlab.com/hmielimo/cloud-resilience-by-default/)
+  - All details regarding storage based Server resiliency incl.
+    - Set up your Server with Resilience by default Using the Console
+    - Set up your Server with Resilience by default using CLI
+    - Performance-Based Auto-tune Use Case script
+- [Block Volume Service performance-based auto-tune - Cost comparison](https://gitlab.com/hmielimo/cloud-resilience-by-default/-/blob/main/doc/Block.Volume.Service.performance.based.auto.tune.Cost.comparison.xlsx)
```

Comment:
Please remove due to contained:
- Oracle pricing details
- Direct comparison vs AWS

Reply:
done